specs/signature-envelope-cose.md

```diff
@@ -13,8 +13,12 @@ Signature Manifest Example
 ```jsonc
 {
     "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
-    "artifactType": "application/vnd.cncf.notary.signature",
-    "blobs": [
+    "config": {
+        "mediaType": "application/vnd.cncf.notary.signature",
```

Comment:
Unclear if this should be the new `scratch`/`empty` config. I'd suggest making a call on the current 1.1.0-rc3 for stability.

Reply:
IMO, the current Notary Project signature manifest is compliant to 1.1.0-rc.3 as a typic artifact according to the [guidelines](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc.3/manifest.md#guidelines-for-artifact-usage)

The `scratch` config was switched to `empty`, which is not released by image spec yet.

Reply:
We should create an issue for this as `v1.1.0-rc4` is out.

Reply:
What is the change required here? Would we use a `empty` config, along with `artifactType` property?

Reply:
@gokarnm I think no changes are required for specifications (notaryproject) v1 release. As Shiwei suggested, I created an [issue ](https://github.com/notaryproject/notaryproject/issues/270) for OCI image spec v1.1.0-rc.4. Regarding to `breaking change`, if you referred to changes required for image spec `v1.1.0-rc.4`, I don't think so, we can discuss further using the this [issue](https://github.com/notaryproject/notaryproject/issues/270)